File: Week_0.1/q12.c
#include <stdio.h>

int main() {
    int i,j,n,a;
    int arr[100];

    printf("Enter the size of an array : ");
    scanf("%d", &n);

    printf("Enter the values of array: ");
    for(i = 0; i < n; i++) {
        scanf("%d", &arr[i]);
    }

    for(i = 0; i < n; i++) {
        for(j = i + 1; j < n; j++) {
            if(arr[i] < arr[j]) {
                a = arr[i];
                arr[i] = arr[j];
                arr[j] = a;
            }
        }
    }

    printf("Sorted array : ");

    for(i = 0; i < n ; i++) {
        printf("%d, ", arr[i]);
    }
    printf("\n");

}

File: Week_3/q12.js
var readlineSync = require('readline-sync')

let limit = readlineSync.question("Enter the size of arrays ")
let a = []

console.log("Enter the values of Array");
for(let i = 0; i < limit; i++) {
    a[i] = Number(readlineSync.question())
}

a.sort((a,b) => b - a)
console.log(`sorted array is \n ${a}`);